### PR TITLE
Added PluribusDriver

### PR DESCRIPTION
**Pull request by mirceaulinic**
Child of #154

### CODE REVIEW

File: napalm/pluribus.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                'remote_port_name'          : port_descr,
+                'remote_system_chassis_id'  : chassis,
+                'remote_system_name'        : system_name
+            }
+
+        return lldp_neighbors
+
+    def show_route(self, destination = ''):
+
+        route = dict()
```
Comment (dbarrosop) [inline napalm/pluribus.py]: Why are you overriding all these methods if they don't do anything? If they are not supported I would rather leave them out so they throw a NotImplementedException.

Comment (mirceaulinic) [inline napalm/pluribus.py]: Will be implemented soon :)

**Comment by dbarrosop** [comment]
Awesome : )

Before merging I have a few questions, it's mostly curiosity, not related to the PR itself:

1. Do you think it will be doable to implement the configuration related parts with time?
2. Do they have a VM or vagrant box?

**Comment by mirceaulinic** [comment]
Unfortunately Pluribus does not provide a VM or Vagrant box, we are running al the tests on physical machines. But we are able to provide mock data.
If you check the library pyPluribus which is extremely simple, at the time being the configuration related parts are not yet deployed but will be ready soon enough to be used in NAPALM.

**Comment by mirceaulinic** [comment]
All right, fixed. Phew!

**Comment by dbarrosop** [comment]
I am afraid tests are not passing:

```
(napalm) ➜  unit git:(mirceaulinic-CF-PLURIBUS) nosetests TestPluribusDriver:TestGetterPluribusriver
EEEEEEEEE
======================================================================
ERROR: test_bgp_neighbors_detail (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 246, in test_bgp_neighbors_detail
    get_bgp_neighbors_detail = self.device.get_bgp_neighbors_detail()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 603, in get_bgp_neighbors_detail
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_bgp_config (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 234, in test_get_bgp_config
    get_bgp_config = self.device.get_bgp_config()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 495, in get_bgp_config
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_bgp_neighbors (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 203, in test_get_bgp_neighbors
    get_bgp_neighbors = self.device.get_bgp_neighbors()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 280, in get_bgp_neighbors
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_environment (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 183, in test_get_environment
    environment = self.device.get_environment()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 302, in get_environment
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_facts (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 150, in test_get_facts
    facts = self.device.get_facts()
  File "/Users/dbarroso/workspace/napalm/napalm/pluribus.py", line 64, in get_facts
    uptime_minutes = int(uptime_minutes_split[0])
ValueError: invalid literal for int() with base 10: '2h42'

======================================================================
ERROR: test_get_interfaces (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 155, in test_get_interfaces
    get_interfaces = self.device.get_interfaces()
  File "/Users/dbarroso/workspace/napalm/napalm/pluribus.py", line 125, in get_interfaces
    'speed'         : speed,
UnboundLocalError: local variable 'speed' referenced before assignment

======================================================================
ERROR: test_get_interfaces_counters (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 174, in test_get_interfaces_counters
    get_interfaces_counters = self.device.get_interfaces_counters()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 369, in get_interfaces_counters
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_lldp_neighbors (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 164, in test_get_lldp_neighbors
    get_lldp_neighbors = self.device.get_lldp_neighbors()
  File "/Users/dbarroso/workspace/napalm/napalm/base.py", line 258, in get_lldp_neighbors
    raise NotImplementedError
NotImplementedError

======================================================================
ERROR: test_get_lldp_neighbors_detail (TestPluribusDriver.TestGetterPluribusriver)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 230, in test_get_lldp_neighbors_detail
    result = result and self._test_model(models.lldp_neighbors_detail, neighbor)
  File "/Users/dbarroso/workspace/napalm/test/unit/base.py", line 134, in _test_model
    same_keys = set(model.keys()) == set(data.keys())
AttributeError: 'str' object has no attribute 'keys'

----------------------------------------------------------------------
Ran 9 tests in 0.002s

FAILED (errors=9)
```

**Comment by mirceaulinic** [comment]
Sorry, forgot to re-run tests after implementing the newest methods. For non-implemented methods I assume it is normal to have errors, isn't it? Please let me know if you see other issues. Thanks! :)

**Pull request merged**